changelog/23118.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+ldaputil: Disable tests for ARM64
```

Comment:
I don't think we need a changelog entry for test only changes

Reply:
oops, think it's worth going back to remove it?

Reply:
Up to you. But I don't think it is necessary

Reply:
Thanks! I'll just leave it